rln/src/protocol.rs

```diff
@@ -350,6 +397,41 @@ pub fn proof_values_from_witness(rln_witness: &RLNWitnessInput) -> Result<RLNPro
     })
 }
 
+pub fn proof_values_from_witness_with_merkle_proof(
```

Comment:
Out of curiosity, is this needed for jswaku? I assume it only uses jswaku can only access what is exposed in rln-wasm?

Reply:
It's not needed, I forgot to remove this after realizing I only needed a new function to generate the witness